### PR TITLE
Add figure for tuning & enrich the tuning section in doc

### LINKED ISSUE #283: Enrich the tuning section of README in DS2

### PR DESCRIPTION
**Pull request by kuke**
Resolve #283

### CODE REVIEW

File: deep_speech_2/README.md [hunk 1]
```diff
@@ -283,28 +283,36 @@ The hyper-parameters $\alpha$ (coefficient for language model scorer) and $\beta
...
+    --beta_to 0.45 \
+    --num_betas 8
     ```
 
 - Tuning with CPU:
 
     ```bash
     python tools/tune.py --use_gpu False
     ```
+ The grid search will log the WER (word error rate) or CER (character error rate) at each point in the hyper-parameter space and their minima, and draw the error surface optionally. A proper hyper-parameters range should include the global minima of the error surface for WER/CER, as illustrated in the following figure.
```
Comment (xinghai-sun) [inline deep_speech_2/README.md]: log-->print ?
remove `and their minima`.

File: deep_speech_2/README.md [hunk 2]
```diff
@@ -283,28 +283,36 @@ The hyper-parameters $\alpha$ (coefficient for language model scorer) and $\beta
...
     ```
+ The grid search will log the WER (word error rate) or CER (character error rate) at each point in the hyper-parameter space and their minima, and draw the error surface optionally. A proper hyper-parameters range should include the global minima of the error surface for WER/CER, as illustrated in the following figure.  
 
-After tuning, you can reset $\alpha$ and $\beta$ in the inference and evaluation modules to see if they really help improve the ASR performance.
+<p align="center">
+<img src="docs/images/tuning_error_surface.png" width=450>
+<br/>An example error surface for tuning on the dev-clean set of LibriSpeech
+</p>
+
+Usually, as the figure shows the variation of language model weight ($alpha$) mainly affect the performance of CTC beam search decoder. And a better procedure is first tuning on serveral data batches (the number can be specified) to find out the proper range of hyper-parameters, then change to the whole validataion set to carray out an accurate tuning.
```
Comment (xinghai-sun) [inline deep_speech_2/README.md]: `as the figure shows` --> `as the figure shows, `
 `is first tuning` --> `is to first tune`
`mainly affect` --> `significantly`
`alpha` --> `$\alpha$`

File: deep_speech_2/README.md [hunk 3]
```diff
@@ -283,28 +283,36 @@ The hyper-parameters $\alpha$ (coefficient for language model scorer) and $\beta
...
+
+After tuning, you can reset $\alpha$ and $\beta$ in the inference and evaluation modules to see if they really help improve the ASR performance. For more help
```
Comment (xinghai-sun) [inline deep_speech_2/README.md]: --> `$\alpha$ and $\beta$

**Review by xinghai-sun — CHANGES_REQUESTED** [review]
Great! Thanks!

File: deep_speech_2/README.md (hunk 1 above)
Comment (kuke) [reply deep_speech_2/README.md]: Done

File: deep_speech_2/README.md (hunk 2 above)
Comment (kuke) [reply deep_speech_2/README.md]: Done

File: deep_speech_2/README.md (hunk 3 above)
Comment (kuke) [reply deep_speech_2/README.md]: Done

**Review by kuke — COMMENTED** [review]
Followed all comments. Thanks!

**Review by xinghai-sun — APPROVED** [review]

**Pull request merged**